### ISSUE TITLE
Bug Report: Message about missing dotnet-desktop is not translated

### ISSUE THREAD

**Issue by bovirus** [Timthreetwelve/GetMyIP#54]
### Is there an existing issue for this?

- [X] I have searched the existing issues (both open and closed).

### What happened or didn't happen?

@Timthreetwelve 

If you try to install GetMyIP ina  Windows system without dotnet-desktiop appear a window that explain the situation and propose to download dotnet-desktop.

In this window the text is untranslated (it's in english).

Please add relative strings in language file.

### Version

0.9.7

### Windows Version

Windows 10 22 H2 64 bit italian

**Comment by Timthreetwelve** [comment]
@bovirus 

The code that detects the missing dotnet desktop runtime is not in my app (any of them). It is most likely built into Windows. So, changing the language in the dotnet dialog isn't an option. (It surprises me that the message is not in the current culture.) 

I've been giving this situation some thought, and I wonder if releasing a variant of the app that has all the necessary dotnet files included would be a solution. In the [.NET documentation](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained) they refer to this as self-contained. This version would be in addition to the current installer and portable versions.

Another option would be to  have the Inno Setup script check to see if the required dotnet runtime environment exists before installing. [This repository](https://github.com/DomGries/InnoDependencyInstaller) looks like what would be needed. You know  Inno Setup better than I do, take a look at it, and let me know what you think.

Thanks,
Tim

**Comment by Timthreetwelve** [comment]
@bovirus 

Version 0.9.8 was released a few minutes ago. I realize that it doesn't directly address your issue. However, with the inclusion of the self-contained builds, I believe that it is a good workaround. 

The addition of the self-contained builds required some updates to the Inno Setup script and the PowerShell script that invokes the Inno Setup compiler. I've added a repo [here](https://github.com/Timthreetwelve/PublishInnoSetup) for the PowerShell script. I added GetMyIPEx.iss which is the script used going forward. 

I'm sorry for the delay and lack of communication recently, I had another health setback and was hospitalized for a few days.  I'm back home now and feeling better. The worst part was the hospital Wi-Fi was down! 

Tim
